mixins/imageHelper.ts

```diff
@@ -83,9 +83,9 @@ const imageHelper = Vue.extend({
         quality: 90
       };
       if (limitByWidth) {
-        params.maxWidth = element.clientWidth.toString();
+        params.maxWidth = Math.round(element.clientWidth * 1.7);
```

Comment:
There's no real need to multiply the image sizes anymore, afaik.

The server folks have improved the scaling a lot. We just hadn't updated old web to match the changes.

It's also very hacky, and I'd prefer we fix any remaining issues with scaling in the server instead of using arbitrary multiplication to "fix" it.

Reply:
Fair enough. My images without this are really blurry, so yeah, we should discuss that with @jellyfin/server